packages/iov-core/src/signingservercore.spec.ts

```diff
@@ -232,6 +232,8 @@ describe("SigningServerCore", () => {
   describe("signAndPost", () => {
     it("can sign and post", async () => {
       pendingWithoutBnsd();
+      // TODO: fix before merge
+      pending("We need to get some money to these accounts somehow.. not sure how");
```

Comment:
You can merge https://github.com/iov-one/iov-core/pull/861 into this to get this done

Reply:
Done